concepts.tex

```diff
@@ -576,16 +611,22 @@
 \pnum
 Let \tcode{a} be an object of type \tcode{T}, \tcode{b} be an object of type \tcode{U}, and \tcode{C} be
 \tcode{\oldtxt{CommonType}}\tcode{\newtxt{common_type_t}}\tcode{<T, U>}. Then \tcode{EqualityComparable<T, U>()}
-is satisfied if and only if
+is satisfied if and only if:
 
 \begin{itemize}
-\item \tcode{bool(a == b) == bool(C(a) == C(b)).}
-\item \tcode{bool(a != b) == bool(C(a) != C(b)).}
-\item \tcode{bool(b == a) == bool(C(b) == C(a)).}
-\item \tcode{bool(b != a) == bool(C(b) != C(a)).}
+\item \tcode{bool(a == b) == bool(C(a) == C(b))}.
+\item \oldtxt{\tcode{bool(a != b) == bool(C(a) != C(b))}.}
+\item \oldtxt{\tcode{bool(b == a) == bool(C(b) == C(a))}.}
+\item \oldtxt{\tcode{bool(b != a) == bool(C(b) != C(a))}.}
 \end{itemize}
 \end{itemdescr}
 
+{\color{newclr}
+\pnum
+\enternote There is no syntactic difference between \tcode{BikeshedEqualityComparable<T, U>()} and
+\tcode{Equal\-ity\-Comp\-arable<\brk{}T, \brk{}U>()}.\exitnote
+} %% color{newclr}
+
```

Comment:
I puzzled over this for a few seconds. Consider saying this as saying the differences are purely semantic.